frontend/pages/_document.tsx

```diff
@@ -0,0 +1,30 @@
+import Document from "next/document";
+import { ServerStyleSheet } from "styled-components";
+
+export default class MyDocument extends Document {
+  static async getInitialProps(ctx) {
```

Comment:
add public here, rest of app is consistent with opting for none implicit access modifiers